changelog/updates/2023-02-22-weekly-updates.md

```diff
@@ -0,0 +1,4 @@
+- SDK: cmake ([3.25.2](https://cmake.org/cmake/help/v3.25/release/3.25.html))
+- iputils ((20221126)[https://github.com/iputils/iputils/releases/tag/20221126])
```

Comment:
Syntax error. It should be `- iputils ([20221126](https://github.com/iputils/iputils/releases/tag/20221126))`.